data/hai/unfettered.txt

```diff
@@ -0,0 +1,271 @@
+		"Remember, you only need to give me a symbolic bribe for this to stop."
+		"This is not a duel to the death, or else only one of us would be able to learn."
+		"Do you wish to surrender?"
+		"I am reasonable. If you wish to surrender, we can come to terms."
+		"You are lucky that I have to accept any ceasefire you ask... you would do well not to attempt anything."
+
+phrase "hostile unfettered challenger"
+	word
+		"Here's a "
+	word
+		"free"
+		"useful"
+		"common"
+	word
+		" tip: "
+		" hint: "
+	word
+		"when feeling overwhelmed in duels you can always ${overwhelmed in duel}! Try it next time."
+		"challengers have common weaknesses, and you should try to get an edge by ${advantage in duel}."
+		
+
+phrase "overwhelmed in duel"
+	word
+		"tactically retreat to the edge of the system"
+		"bribe the challengers"
+		"try to split the enemies apart"
+		"go to another system"
+		"upgrade your ship design"
+		"scan the enemy to find out their vulnerabilities"
+
+phrase "advantage in duel"
+	word
+		"figuring out if their engines are weak"
+		"trying to slide away whilst shooting them back"
+		"using your range advantage, be it in missiles, Ion Cannons or else"
+		"try to get your hands on powerful prototypes"
+		"going and acquiring alien technology. Even human tech has its uses"
+		"overheat their ship"
+		"shoot them down with missiles if they lack sufficient jammers and anti-missiles"
+		"try to get at their batteries with ion weapons"
+
+phrase "hostile unfettered challenger"
+	word
+		"If you happen to know of "
+		"If you heard of "
+		"When you come to know about "
+		"Next time you get your hands on "
+		`If you "find" `
+	word
+		"a strong"
+		"an interesting"
+		"a good"
+		"an exotic"
+		"a respectable"
+		"a powerful"
+		"an overpowered"
+	word
+		" weapon combo"
+		" alien weapon"
+		" human junk-weapon"
+	word
+		", you should come and "
+	word
+		"demonstrate it to us in action"
+		"fight me with it"
+		"sell it to us, after a proper demonstration"
+	word
+		"."
+	word
+		" You might find some of us willing to buy it, for the proper price"
+	word
+		"!"
+
+phrase "defeated unfettered challenger"
+	word
+		"You have "
+		"You managed to have "
+		"Somehow, you have "
+		"I must admit you have "
+	word
+		"beaten "
+		"defeated "
+		"ashamed "
+		"disabled "
+		"revealed my weakness% "
+	word
+		"me in a% "
+	word
+		"glorious "
+		"pathetic "
+		"exceptional "
+		"grandiose "
+		"fast-paced "
+		"strategic "
+	word
+		"battle"
+		"combat"
+		"duel"
+		"fight"
+		"challenge"
+	word
+		"."
+		"!"
+	word
+		" What more do you want from me?" 3
+		" Do not presume you may board me." 3
+		" Do not even think about capturing my ship." 3
+		" Now that it is over, time to lay down your weapons." 3
+		" I accept my fate. Take what is rightfully yours." 3
+		" Great stories will be told!" 3
+		" You have the right to plunder." 3
+		" It would seem you are as strong as some claim, perhaps even more." 3
```

Comment:
" It would seem you are as strong as some claim, perhaps even more." -> " It would seem you are as strong as some claim, perhaps even more so."

Reply:
```suggestion
		" It would seem you are as strong as some claim, perhaps even more so." 3
```